samples/endpoint/endpoint_config.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
```

Comment:
I think we discussed this, but can we not include all 3 of these  in a single file ? 
Something like this - [HAP Sample](https://github.com/aws-controllers-k8s/applicationautoscaling-controller/blob/main/samples/hosting-autoscaling-on-sagemaker/hosting-autoscaling-predefined.yaml)